File: fizzbuzz.cs
using static System.Console;

class FuzzBuzz 
{
	static void doFizzBuzz(int num1, int num2){
		bool fizz;
		bool buzz;
		
		for (int i = 1; i <= 100; i++){
			fizz = (i%num1==0);
			buzz = (i%num2==0);
			
			if(fizz && buzz)
    			WriteLine("FizzBuzz");
    		
			else if(fizz)
    			WriteLine("Fizz");
    		
			else if(buzz)
    			WriteLine("Buzz");
    		
			else 
    			WriteLine(i);
		}
	}
	static void Main() {
		doFizzBuzz(3,5);
	}
}

File: fizzbuzz.js
function doFizzBuzz(num1, num2){
	    var fizz;
	    var buzz;
	for(var i = 1; i < 101; i++){
		fizz = (i%num1==0);
		buzz = (i%num2==0);
		if(fizz && buzz){
			console.log("FizzBuzz");
		}
		else if(fizz){
			console.log("Fizz");
		}
		else if(buzz){
			console.log("Buzz");
		}
		else {
			console.log(i);
		}
	}
}

doFizzBuzz(3, 5);
